src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -233,7 +267,7 @@ protected [db] class CampaignStatusTransition(implicit db: Database, ec: Executi
     val countFinishedDevices =
```

Comment:
Something is odd. If we make this modification, then `val countFinishedDevices` is misleading, because "finished" is defined as "either successful or failed", as in the method `countFinished` up in this file. I feel either this modification is not needed or the name should be changed, but I'm not sure...

Reply:
This modification is coming from #98, so I'll move your comment to that PR.